docs/UserGuide.md

```diff
@@ -180,8 +180,7 @@ Format: `addClaim INDEX iid/INSURANCE_ID cid/CLAIM_ID ca/CLAIM_AMOUNT_IN_DOLLARS
 <br>
 2. `INSURANCE_ID` : Must be an existing valid ID that is currently supported.
 <br> Currently supported IDs for insurance plans: `Basic Insurance Plan` : `0` & `Travel Insurance Plan`: `1`. <br>
-3. `CLAIM_ID` : a unique id for each claim generated by the company and should be created externally by another
-organisation before the claim is added to this app.
+3. `CLAIM_ID` : a unique id for each claim generated by an external organisation before the claim is added to this app.
```

Comment:
Maybe put `.. generated by the insurance company or an external organisation (eg. law firm)` to be clearer? In case they were to point out in the PE that `external organisation` is not clear...

Reply:
Okie